packages/convenient_test_common/lib/src/common_flutter/non_ballistic_clamping_scroll_physics.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:math';
 import 'package:convenient_test_common_dart/convenient_test_common_dart.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
+// ignore: unnecessary_import
```

Comment:
Just remove the unnecessry import :)

Reply:
Unfortunately I can't. When using Flutter 3.0, this import is requried, but on Flutter 3.3 (and beta as well, ofc) it's detected as "unnecessary" because some other package started re-exporting it.

Reply:
You are right!